src/components/Deposit.tsx

```
@@ -0,0 +1,58 @@
+      </button>
+      <dialog ref={dialogRef} className="modal">
+        <div className="modal-box">
+          <h3 className="text-lg font-bold">Add new validator</h3>
+          <p className="text-sm text-gray-500">Balance: {Number(formatEther(balance.balance)).toFixed(2)} GNO</p>
+          <fieldset className="fieldset">
+            <legend className="fieldset-legend">Upload deposit data file</legend>
+            <input type="file" className="file-input" onChange={handleFileChange} />
+            <label className="label">{file?.name}</label>
+          </fieldset>
+          <div className="mt-8 flex w-full justify-end">
+            <button className="btn btn-primary" disabled={0 === 0} onClick={() => { }}>
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)